Adding 1Password for Windows as an FMA

tux234 opened the pull request:
Adding the Windows version of 1Password to our Fleet Maintained App Catalog.

tux234 committed:
Adding 1Password for Windows as an FMA

allenhouchins commented:
Moving back to Draft until the issue is resolved. 

```
level=info ts=2025-11-17T17:06:28.8355847Z caller=windows.go:77 app=1Password msg="Found app: '1Password' at C:\\Program Files\\1Password\\, Version: 8.11.18"
level=error ts=2025-11-17T17:06:28.8355847Z caller=main.go:176 app=1Password msg="App version '8.11.18.36' was not found by osquery"
```

tux234 committed:
Fixed verionsing mismatch in 1Password/windows.json

tux234 committed:
Add version transformer for 1Password Windows

Following the pattern as mentioned by @allenhouchins. Created Windows version of external refs as well as appropriate functions to transform 1Password version from 8.11.18.36 to 8.11.18.

allenhouchins committed:
Merge branch 'main' into tux234-add-fma-windows-1password
